### DISCUSSION TITLE
v0.8.5 - Typing

### DISCUSSION THREAD

**Discussion by awtkns** [awtkns/fastapi-crudrouter#140] [General]
### 🎉 Highlights
With the release of v0.8.5 fastapi-crudrouter now officially supports both **Python 3.10** and **typed python**. This release also includes significant improvements to the documentation, test suite, and developer experience. 

Keep an eye of for the next release which will contain support for **async SQLAlchemy** (#122). 

Big thanks to all contributors that made this possible!

### ✨ Features
- Typed python support #132 #111
- Python 3.10 support #114
- Test suite now runs against multiple databases #86
- Documentation improvements #79 #91 #117 #123 #124 #125 @andrewthetechie
- More informative exceptions #94 #137
- General test suite improvements #96 #97

### 🐛 Bug Fixes
- OrderBy not working correctly with Microsoft SQL Server #88
- 404 response not documented in OpenAPI spec #104 @sondrelg
- DatabasesCRUDRouter not functioning for inserts and deletes with AsyncPG #98

**Full Changelog**: https://github.com/awtkns/fastapi-crudrouter/compare/v0.8.0...v0.8.5

<hr /><em>This discussion was created from the release <a href='https://github.com/awtkns/fastapi-crudrouter/releases/tag/v0.8.5'>v0.8.5 - Typing</a>.</em>

**Comment by MatthijsKok** [comment]
Thank you so much for this release, keep it up!  
Looking forward to the async support (and possibly SQLModel in the future? 😮)

**Reply by awtkns** [reply]
Hi @MatthijsKok thanks for the feedback!

Although not documented, SQLModel is actually already somewhat supported as it relies on SQLAlchemy. Check out #108 if you're interested.  🚀
